src/components/collection-component/load-data.js

```diff
@@ -1,36 +1,54 @@
-const collection1 = document.querySelector('#collection-1')
-const collection2 = document.querySelector('#collection-2')
-const collection3 = document.querySelector('#collection-3')
+const collection1 = document.querySelector('#collection-1');
+const collection2 = document.querySelector('#collection-2');
+const collection3 = document.querySelector('#collection-3');
 
+const best2021 = 'LS_BEST_2021';
```

Comment:
co to jest?

Reply:
ja to pobrałem z merge od Judyty na mian i zostawiłem jej to jak rozwiązywałem konflikt żeby sie nie okazało że to jest jej do czegoś potrzebne ;) ale to może być praktyka zaczerpnieta z kluczy do useReducer w react że jak sobie przypiszesz string do zmiennej to masz pewnośc że nie zrobisz literówki w użyciu i łatwiej wyszukać błąd ;)